SEED-platform/seed
# Added `COOKIE_EXPIRATION` environment variable

#### Any background context you want to provide?
We were using `SESSION_EXPIRE_AT_BROWSER_CLOSE`, which in "normal" cases causes the user to be logged out every time they close their browser. However, in modern browser environments with Chrome and Firefox most users have their browsers set to resume their session upon reopening, which causes sessions to be valid until the CSRF cookie expires (**1 year**).

#### What's this PR do?
Adds support for a `COOKIE_EXPIRATION` environment variable that limits cookie validity to the Django default of two weeks, unless overridden

#### How should this be manually tested?
1. Logout/Login
2. Check the `Application/Cookies` section of the browser's developer tools, and ensure that the expiration is two weeks from now instead of `Session`

#### What are the relevant tickets?
#4800
